### PR TITLE
Translate some text into English

### PR DESCRIPTION
**Pull request by xfq**

### CODE REVIEW

File: index_en.html [hunk 1]
```diff
@@ -318,6 +335,32 @@ <h2>Interactive wall</h2>
         In this kind of scene, the Bullet Chatting is also displayed separately, not attached to the video.
       </p>
     </section>
+    <section>
+      <h2>Masking</h2>
+      <p>
+        <p>
+          When watching a video, there may be a large number of bullet chatting appearing on the screen at the same time, causing the content of the video itself to be covered, which will affect watching the media content. For example, when the user is learning to dance, they may not see the movement of the dancer. [To be translated part: 这种情况下，就可以借助计算机视觉和AI的能力，对视频内容进行分析，并将之前已经定义好的“视频主体内容”进行识别，生成蒙版并分发给客户端后，客户端利用CSS3的特性进行渲染从而达成弹幕不会覆盖定义的“视频主体内容”的效果，这种弹幕被称为“蒙版弹幕”，也被称为“防挡弹幕”。]
```
Comment (siusin) [inline index_en.html]: s/causing the content of the video itself to be covered, which will affect watching the media content/which may unexpectedly cover the video itself, and disrupt users from watching the media content

File: index_en.html [hunk 2]
```diff
@@ -333,9 +376,12 @@ <h2>Examples</h2>
           </pre>
       </aside>
       <p class="note">
-        Bullet Chatting make use of new components like
-        <code>bulletchatlist</code>和<code>bulletchat</code>，and they are not inherited from
-        <code>video</code> or <code>audio</code>.
+        Bullet Chatting uses new elements such as
+        <code>bulletchatlist</code> and <code>bulletchat</code>. These elements are not used as children of the media elements <code>audio</code> and <code>video</code>. From the Scenarios section, we can see that bullet chatting is not only related to media elements such as video. In scenes like Web page interaction or interactive wall, bullet chatting is independent to any media element.
```
Comment (siusin) [inline index_en.html]: s/we can see that bullet chatting is not only related to media elements such as video/we can see that the usage of bullet chatting is not limited to those media elements like video

File: index_en.html (hunk 1 above)
Comment (xfq) [reply index_en.html]: Fixed: https://github.com/w3c/danmaku/pull/14/commits/def550f751355c4c16ffc31307e2d245f3a23788

File: index_en.html (hunk 2 above)
Comment (xfq) [reply index_en.html]: Fixed: https://github.com/w3c/danmaku/pull/14/commits/def550f751355c4c16ffc31307e2d245f3a23788

**Pull request merged**